test/helpers/replay-setup.js

```diff
@@ -9,4 +9,5 @@ const Replay = require('replay')
 
 module.exports = subFolder => {
   Replay.fixtures = path.join(__dirname, '..', 'http-mocks', subFolder)
+  Replay.headers = [/^accept/, /^body/, /^content-type/, /^host/, /^if-/, /^x-/]
```

Comment:
not sure what `^body` is?

Reply:
It's what `Replay.headers` already contained, I've gone ahead and copied it and removed the authorization regex. I assume it's how Replay sees the request body.